cellxgene_schema_cli/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to the python package `cellxgene-schema` are documented in t
 The format of this changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.3] - 2021-10-08
+### Changed
+- Replace print with logging
+- Return the list of errors from validate so they can be used by callers
```

Comment:
- Replace print with logging in `Validator` and `AnnDataLabelAppender`.
- Return the list of errors from `validate` so they can be used by callers.
- Add verbose option to CLI, it prints a progress log.